infra/src/environments.ts

```diff
@@ -3,6 +3,8 @@ import { EnvironmentConfigs } from './dsl/types/charts'
 export const Envs: EnvironmentConfigs = {
   dev: {
     auroraHost: 'postgres-applications.internal',
+    auroraReplica:
+      'dev-vidspyrna-aurora.cluster-ro-c6cxecmrvlpq.eu-west-1.rds.amazonaws.com',
```

Comment:
Is this safe to configure in all the monorepo charts? Should judicial-system use the same aurora clusters as islandis?

Reply:
Yeah it should be safe, we have one aurora cluster for all of our applications (judicial-system included) and the plan is to keep on doing that for additional applications.